### PR TITLE
FIX: Fixed minor brand issues

### PR DESCRIPTION
**Pull request by stunthamster**
**INTENT**
Fix minor typos in brand

**DETAIL**
ONIE and mion are brands, and should always be written with those respective cases (Unless, in the case of mion, its starting a sentence). This PR corrects those. Also, I seem to have a plugin that strips dangling white space, so that's happened as well.

### CODE REVIEW

File: docs/installing_mion.md [hunk 1]
```diff
@@ -11,26 +11,26 @@ Installing Mion
 
 Introduction
 -------
-The Mion 0.9 release currently supports a single platform - the **APS Networks (Formally Stordis) BF2556X-1T** 
+The Mion 0.9 release currently supports a single platform - the **APS Networks (Formally Stordis) BF2556X-1T**
```
Comment (CommitThis) [inline docs/installing_mion.md]: You need to change to "formerly" here

**Review by CommitThis — CHANGES_REQUESTED** [review]
Jus one comment on a missed spelling mistake

File: docs/installing_mion.md (hunk 1 above)
Comment (stunthamster) [reply docs/installing_mion.md]: Good catch

**Comment by kprosise** [comment]
Just a reminder that pull requests with multiple commits need to be rebased, in line with the workflow of the other mion repos, though I'll approve this one as I don't believe  squashing commits has been communicated clearly in the workflow info.

**Pull request merged**